docs/en/index.md

```diff
@@ -1,48 +1,15 @@
-title: Login Forms
+---
+title: Login forms
 summary: Simplifying the login forms for a SilverStripe site
+icon: file-lines
```

Comment:
This icon doesn't exist in font awesome v5: https://fontawesome.com/v5/search?q=file-lines&o=r
You may have meant `file-alt`: https://fontawesome.com/v5/search?q=file-alt&o=r

Reply:
Updated